pmlc/rt/jit_executable.cc

```diff
@@ -400,18 +400,21 @@ class JitExecutable final : public Executable {
     }
   }
 
-  void invoke(mlir::ArrayRef<util::BufferPtr> inputBuffers,
-              mlir::ArrayRef<util::BufferPtr> outputBuffers) final {
+  double invoke(mlir::ArrayRef<util::BufferPtr> inputBuffers,
+                mlir::ArrayRef<util::BufferPtr> outputBuffers) final {
     StopWatch stopWatch;
-    if (VLOG_IS_ON(1)) {
-      stopWatch.start();
-    }
+    stopWatch.start();
     bindArguments(inputBuffers, outputBuffers);
     jitEntry(ptrs.data());
-    if (VLOG_IS_ON(1)) {
-      stopWatch.stop();
-      IVLOG(1, "Execution time: " << stopWatch.delta_ms() << "ms");
+    stopWatch.stop();
+
+    double execTime = device->execTimeInMS;
+    if (execTime == 0.0) {
```

Comment:
@flaub please check my logic here.  by doing this, I *always* return a non-zero execution time.  which means that even for pipelines such as CPU which do not set device->execTimeInMS we will use this "inner" stopwatch time to override `exec_per_example` in plaidbench.  not sure if that's what we want.  alternatively, I can move this check-for-zero to frontend_keras.py and only set the override in the non-zero case.  in that case I would simply return device->execTimeInMS from this function.

Reply:
the latest commit moves check-for-zero to frontend_keras.py and sets override accordingly.  I think this is what we want.